### DISCUSSION TITLE
How do I release to play store open beta track instead of closed beta?

### DISCUSSION THREAD

**Discussion by lotdrops** [fastlane/fastlane#20108] [Q&A]
In the play store I have an internal track, a closed alpha, a closed beta, an open beta, and production. All of these match the existing supply tracks **except the open beta**:
The supply track "beta" uploads to the closed beta channel in the play store. However, I want to upload to the open beta channel and I do not know **how can I find out its track name** to use in supply.

**Comment by obiwan007** [comment]
Any news on this item?
I can not release to any beta track as they seems to be not existing in the usual naming...
